go/libkb/identify_outcome.go

```diff
@@ -144,8 +144,11 @@ func (i IdentifyOutcome) NumTrackChanges() int {
 }
 
 func (i IdentifyOutcome) TrackStatus() keybase1.TrackStatus {
-	if i.NumTrackFailures() > 0 || i.NumRevoked() > 0 {
-		return keybase1.TrackStatus_UPDATE_BROKEN
+	if i.NumTrackFailures() > 0 {
```

Comment:
I believe you want to reverse the order of these two, and have the revoked proofs take precedence over the failed proofs.  This way if you have both revoked and failing proofs, you won't be given the opportunity to snooze.